### DISCUSSION TITLE
Unable to parse --filters parameters

### DISCUSSION THREAD

**Discussion by Danny-m08** [aws/aws-cli#7127] [Q&A] [answered]
I cannot figure out what I am doing wrong in the following command:

```
aws ec2 describe-instances --filters  Name=tag:tag1,Values="1" --query Reservations[*].Instances[*].[InstanceId] --output=text
Error parsing parameter '--filters': Expected: '=', received: 'N' for input:
 Name=tag:tag1,Values="1"
 ^
```

**Comment by aBurmeseDev** [comment, marked as answer]
Hi @Danny-m08 , thanks for reaching out.
I couldn't reproduce the error, though I'm fairly certain that the root causes for these errors seem to be the misuse of single/double quotes or spacing in some cases. I'd suggest to double check any extra spaces in your command and check out these posts for references [here](https://stackoverflow.com/questions/62767563/filtering-by-tags-not-working-when-using-aws-ec2-describe-instances-from-command) and [here](https://github.com/aws/aws-cli/issues/2201).

Here's the command I ran and it outputs without errors:
```
aws ec2 describe-instances --query "Reservations[*].Instances[*].{Name:Tags[?Key=='Name']|[0].Value,Status:State.Name}" --filters "Name=tag:Name,Values='*'"  --output table
```

Please let me know if any of this resolves the issue!
